src/main/java/org/ow2/proactive/catalog/service/exception/ParsingObjectException.java

```diff
@@ -32,11 +32,17 @@
 /**
  * @author ActiveEon Team
  */
-@ResponseStatus(value = HttpStatus.UNSUPPORTED_MEDIA_TYPE)
-public class UnsupportedMediaTypeException extends ResourceNotFoundException {
+@ResponseStatus(value = HttpStatus.UNPROCESSABLE_ENTITY)
+public class ParsingObjectException extends RuntimeException {
 
-    public UnsupportedMediaTypeException(String message) {
+    final public static String ERROR_MESSAGE = "It was not possible to parse an object: ";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)